File: 519b/main.go
package main

import "fmt"

func main() {
	var n int
	var sum [3]int64

	fmt.Scan(&n)

	var v int64
	for i := 0; i < n; i++ {
		fmt.Scan(&v)
		sum[0] += v
	}

	for i := 0; i < n-1; i++ {
		fmt.Scan(&v)
		sum[1] += v
	}

	for i := 0; i < n-2; i++ {
		fmt.Scan(&v)
		sum[2] += v
	}

	fmt.Printf("%d\n%d\n", sum[0]-sum[1], sum[1]-sum[2])
}

File: 519b/main.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    long long v;
    long long sum[3] = {0};
    for (int i = 0; i < n; i++) {
        cin >> v;
        sum[0] += v;
    }
    for (int i = 0; i < n - 1; i++) {
        cin >> v;
        sum[1] += v;
    }
    for (int i = 0; i < n - 2; i++) {
        cin >> v;
        sum[2] += v;
    }

    cout << sum[0] - sum[1] << endl 
         << sum[1] - sum[2] << endl;
}
